### ISSUE TITLE
stylelint-config-standard doesn't build anymore

### ISSUE THREAD

**Issue by illuusio** [marguerite/nodejs-packaging#19]
stylelint-config-standard doesn't build anymore and it produces this kind error:
```
[   18s] + node2rpm-server --mkdir
[   19s] /usr/lib64/ruby/gems/2.1.0/gems/node2rpm-1.0.0/lib/node2rpm/server.rb:265:in `block in manual_requires': undefined method `[]' for nil:NilClass (NoMethodError)
[   19s] 	from /usr/lib64/ruby/gems/2.1.0/gems/node2rpm-1.0.0/lib/node2rpm/server.rb:264:in `each'
[   19s] 	from /usr/lib64/ruby/gems/2.1.0/gems/node2rpm-1.0.0/lib/node2rpm/server.rb:264:in `manual_requires'
[   19s] 	from /usr/lib64/ruby/gems/2.1.0/gems/node2rpm-1.0.0/lib/node2rpm/server.rb:278:in `block in recursive_mkdir'
[   19s] 	from /usr/lib64/ruby/gems/2.1.0/gems/node2rpm-1.0.0/lib/node2rpm/server.rb:274:in `each'
[   19s] 	from /usr/lib64/ruby/gems/2.1.0/gems/node2rpm-1.0.0/lib/node2rpm/server.rb:274:in `recursive_mkdir'
[   19s] 	from /usr/lib64/ruby/gems/2.1.0/gems/node2rpm-1.0.0/lib/node2rpm/server.rb:36:in `mkdir'
[   19s] 	from /usr/lib64/ruby/gems/2.1.0/gems/node2rpm-1.0.0/bin/node2rpm-server:13:in `<top (required)>'
[   19s] 	from /usr/bin/node2rpm-server:23:in `load'
[   19s] 	from /usr/bin/node2rpm-server:23:in `<main>'
[   19s] error: Bad exit status from /var/tmp/rpm-tmp.wAlyO2 (%install)
```
Package can be found from: https://build.opensuse.org/package/show/home:illuusio:nodejs-bundle/stylelint-config-standard

**Comment by marguerite** [comment]
The manual_requires function supposed the manual Requires are like this:

    Requires: npm(xxx) >= 1.0.0

I didn't consider packagers may want to Requires stuff other than a npm module.

Just add some escape stuff and testing.

BTW, @illuusio why you Requires the name that the package itself provides. like A requires A, which is redundant and wrong...but of course the issue you found is still valid, because packagers may want to requires like this "Requires: bash"

**Comment by marguerite** [comment]
delivered: https://github.com/marguerite/rubygem-node2rpm/commit/51d133038dd8a7c48e34c4fa205a3e35065e874f

closed #19

**Comment by illuusio** [comment]
Sorry being n00b with nodejs packaging and using. Nice catch indeed.. It's made automatic tool provided (Nova packager of that new one) so it's inside package.json of that package.. I'll remove. Thanks again fast solving!
